Source/Assets/AppCenterEditorExtensions/Editor/Scripts/Utils/EdExLoggerFactory.cs

```diff
@@ -0,0 +1,25 @@
+﻿
+//Factory.
+namespace AppCenterEditor
+{
+    class EdExLoggerFactory
```

Comment:
Why class is called factory whereas in reality it is Singleton? :)

Reply:
Because it provides instance of Logger, hence `Factory`. It is not exactly a singleton - it just stores `Logger` instance